src/rmp/include/rmp/blif.h

```diff
@@ -85,6 +92,8 @@ class Blif
   std::string const0_cell_port_;
   std::string const1_cell_;
   std::string const1_cell_port_;
+  std::map<std::string, std::pair<float,float>> requireds_;
+  std::map<std::string, std::pair<float,float>> arrivals_;
```

Comment:
Why are you using strings rather than object pointers?

Reply:
Writer is independent of db pointers right now. So working with just names everywhere while writing Blif